app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume_snapshot.rb

```diff
@@ -81,7 +81,17 @@ def delete_snapshot_queue(userid = "system", _options = {})
   def delete_snapshot(_options = {})
     with_provider_object do |snapshot|
       if snapshot
-        snapshot.delete
+        begin
+          snapshot.delete
+        rescue => e
+          notification_options = {
+            :snapshot_op => 'delete',
+            :subject     => "[#{name}]",
```

Comment:
This is what message looks like if I use self for subject: 
<img width="317" alt="screen shot 2017-06-19 at 14 53 05" src="https://user-images.githubusercontent.com/9213557/27534458-5625fe62-5a67-11e7-82be-7d9bb70f294f.png">

So for now I left  ``"[#{name}]"``